bin/install/install-dcos-cli.sh

```diff
@@ -4,7 +4,34 @@ set -o errexit -o pipefail
 
 usage()
 { # Show usage information.
-  echo "$(basename "$(test -L "$0" && readlink "$0" || echo "$0")") <installation-path> <dcos-url>"
+  echo "$(basename "$(test -L "$0" && readlink "$0" || echo "$0")") <installation-path> <dcos-url> [--add-path yes/no]"
+}
+
+add_dcos_path_to_profile()
+{
+    UNAME=`uname`
+    case "$UNAME" in
+        Linux ) RC_NAME=".bashrc";;
+        Darwin ) RC_NAME=".bash_profile";;
+        CYGWIN* ) RC_NAME=".bashrc";;
+        MINGW* ) RC_NAME=".profile";;
+        * ) RC_NAME=".bashrc";;
+    esac
+    echo "" >> ~/"$RC_NAME"
+    echo "# add the path to the DCOS binary to your bash profile" >> ~/"$RC_NAME"
+    echo "export PATH=$1:\$PATH" >> ~/"$RC_NAME"
+}
+
+prompt_add_dcos_path_to_profile()
+{
+    while true; do
+        read -p "Do you wish to add the path to the DCOS binary to your bash profile? [yes/no] " ANSWER
```

Comment:
`"Do you wish to add the path to the DCOS binary to your bash profile? [yes/no] "` -> `Modify your bash profile to add DCOS to your PATH? [yes/no] `